public/js/main.js

```diff
@@ -37,10 +34,14 @@ const createStore = configureStore({
 const store = createStore(combineReducers(reducers));
 const actions = bindActionCreators(require("./actions"), store.dispatch);
 
-// global for debugging purposes only!
-window.store = store;
-window.injectDebuggee = require("./test/utils/debuggee");
-window.serializeStore = () => JSON.parse(JSON.stringify(store.getState()));
+if (isEnabled("development")) {
+  AppConstants.DEBUG_JS_MODULES = true;
+  debugGlobal("store", store);
+  debugGlobal("injectDebuggee", require("./test/utils/debuggee"));
+  debugGlobal("serializeStore", () => {
+    return JSON.parse(JSON.stringify(store.getState()))
+  });
+}
```

Comment:
I was hoping to move all of this out of main.js and into the debug module. Something like `injectGlobals` that would be called here (you'd need to pass what you want to inject).